docs/installation-mac.md

```diff
@@ -26,7 +26,7 @@ sh -c "$(curl -fsSlL https://github.com/observiq/observiq-otel-collector/release
 
 ### Additional Install Steps
 
-If you plan on JMX collecting metrics via the [JMX Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.49.0/receiver/jmxreceiver/README.md) you can copy the `opentelemetry-java-contrib-jmx-metrics.jar` to the default location to make configuration easier.
+If you plan on JMX collecting metrics via the [JMX Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.50.0/receiver/jmxreceiver/README.md) you can copy the `opentelemetry-java-contrib-jmx-metrics.jar` to the default location to make configuration easier.
```

Comment:
nit and won't hold up PR for this but would be better if it was like "If you plan on collecting metrics via that [JMX Receiver]..."